src/server/cluster/incoming_slot_migration.h

```diff
@@ -21,8 +21,13 @@ class IncomingSlotMigration {
   ~IncomingSlotMigration();
 
   // process data from FDLYMIGRATE FLOW cmd
+  // executes until Cancel called or connection closed
   void StartFlow(uint32_t shard, util::FiberSocketBase* source);
+
   // wait untill all flows are got FIN opcode
+  // Join can be called several times and connection isn't closed
```

Comment:
```suggestion
  // Join can be called several times while the connection is still open
```
I'm not sure if this is what you meant to say in the comment? If not, can you kindly rephrase?